packages/typescript/src/options/tsconfig.ts

```diff
@@ -16,6 +25,19 @@ import {
 } from './interfaces';
 import { normalizeCompilerOptions, makePathsAbsolute } from './normalize';
 
+export interface TypeScriptConfig {
+  autoSetSourceMap: boolean;
+  options: CompilerOptions;
+  typeAcquisition?: TypeAcquisition | undefined;
```

Comment:
It can already be `undefined` from the `?:`

```suggestion
  typeAcquisition?: TypeAcquisition;
```

Reply:
This was generated by tsc. That means the property can be there or not, and if it's there it can be a value or undefined. I think that's different than how you're interpreting that.